tests/.eslintrc.js

```diff
@@ -1,9 +1,13 @@
+/* eslint sort-keys: 'error' */
+
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
   extends: ['../.eslintrc.js'],
   rules: {
-    'mocha/max-top-level-suites': 0,
-    'func-names': 0,
-    'mocha/no-hooks-for-single-case': 0,
+    'func-names': 'off',
+    'local-rules/semantic-mocha/no-member-only': 'error',
+    'local-rules/semantic-mocha/no-member-skip': 'error',
```

Comment:
Should these rules default to error?

Reply:
This is the place where you set the severity of a rule. Rules do not have an intrinsic severity level.